lib/features/purchase/domain/entities/payment_status.dart

```diff
@@ -32,12 +32,12 @@ enum PaymentStatus {
   @override
   String toString() {
     return switch (this) {
-      PaymentStatus.completed => 'Completed',
-      PaymentStatus.rejectedPayment => 'Rejected',
-      PaymentStatus.awaitingPayment => 'Pending',
-      PaymentStatus.refunded => 'Refunded',
-      PaymentStatus.error => 'Error',
-      PaymentStatus.reserved => 'Reserved',
+      PaymentStatus.completed => 'Purchased',
```

Comment:
perhaps these should be in Strings.dart

Reply:
Might agree, but out of scope of this PR

Reply:
I would prefer if this was fixed now while we are fixing stuff here